src/rpc/mempool.cpp

```diff
@@ -547,6 +645,40 @@ static RPCHelpMan getmempooldescendants()
     };
 }
 
+static RPCHelpMan getmempoolcluster()
+{
+    return RPCHelpMan{"getmempoolcluster",
+        "\nReturns mempool data for given cluster\n",
+        {
+            {"id", RPCArg::Type::NUM, RPCArg::Optional::NO, "The cluster id (must be in mempool)"},
```

Comment:
How are users of the RPC expected to obtain `cluster_id`? 

IIUC, the way to do this currently is to query 

```
./bin/bitcoin-cli getrawmempool true
```

and then aggregate the `cluster_id`'s yourself?

Reply:
Sjors suggested here https://github.com/bitcoin/bitcoin/pull/28676#discussion_r1399137900 that I add a way to do it by transaction hash, which would be easy. Let me know if you have other suggestions...